tpkg/ede.tdir/ede.conf

```diff
@@ -0,0 +1,16 @@
+server:
+    logfile: "nsd.log"
+    difffile: ede.ixfr.db
+    xfrdfile: ede.xfrd.state
+    zonesdir: ""
+    hide-version: yes
+    zonelistfile: "zone.list"
+    interface: 127.0.0.1
+
+zone:
+    name: example.com.
+    zonefile: ede.zone
+
+zone:
+    name: example.net.
+    zonefile: does-not-exsist
```

Comment:
This zonefile: parameter can also be empty with `""`, and then this misspelled entry can be fixed (or change `s/exsist/exist/`.

Reply:
resolved in commit fcafc1a